Makefile

```diff
@@ -3,33 +3,18 @@
 # SPDX-License-Identifier: MPL-2.0
 ts := $(shell date +"%Y%m%d%H%M")
 
-binary:
-		nix-build -A tezos-client-babylonnet -o tezos-client
-		cp tezos-client/tezos-client-babylonnet-* tezos-client-babylonnet
-		rm tezos-client
+binaries:
+		nix-build -A babylonnet-binaries -o binaries
+		cp binaries/* ./
+		rm binaries
```

Comment:
AFAIU `binaries` is a directory and it is likely not empty. Will it work w/o `-r`?

Reply:
At this point `binaries` directory has only one `.tar.gz` archive inside. Everything is fine for me locally:
```
XPS-15-9560 :: work/serokell/tezos-client-packaging » make binaries
nix-build -A babylonnet-binaries -o binaries
/nix/store/dn4cmd7m03qsq6ijzq0lhklb3ch6y01f-tezos-babylonnet-binaries-b8731913.tar.gz
cp binaries/* ./
rm binaries
XPS-15-9560 :: work/serokell/tezos-client-packaging » echo $?
0
```

Reply:
> This manual page documents the GNU version of rm.  rm removes each specified file.  By default, it does not remove directories.

Strange :man_shrugging:

Reply:
ah, I misunderstood you, at this point `binaries` is actually a symlink to some path in `/nix/store`

Reply:
Ah, coolstory, ok then.